src/Page/Index.elm

```diff
@@ -75,3 +90,29 @@ teaserColumnStyle =
         , withMediaTabletPortraitUp [ flexDirection column ]
         , withMediaMobileUp [ flexDirection row ]
         ]
+
+
+callForStoryStyle : Style
+callForStoryStyle =
+    batch
+        [ backgroundColor purple
+        , color white
+        , padding (rem 1)
+        , width (pct 100)
+        ]
+
+
+callForStoryHeadingStyle : Style
+callForStoryHeadingStyle =
+    color white
+
+
+callForStoryLinkStyle : Style
+callForStoryLinkStyle =
+    batch
+        [ backgroundImage (url "/images/arrow--white.svg")
+        , backgroundPosition right
+        , backgroundRepeat noRepeat
+        , color white
```

Comment:
```suggestion
        , color white
        , fontWeight (int 500)
```

I know it's not Emma's design but it would be good if this link was differentiated from regular text